core/src/test/java/org/apache/druid/math/expr/ExprEvalTest.java

```diff
@@ -143,10 +151,10 @@ public void testLongArrayEvalTooBig()
     expectedException.expectMessage(StringUtils.format(
         "Unable to serialize [%s], size [%s] is larger than max [%s]",
         ExpressionType.LONG_ARRAY,
-        NullHandling.sqlCompatible() ? 33 : 30,
+        14,
```

Comment:
Is this the length? If so, we've more than halved the number of values which can be stored. Is this a good thing? Or, did we force-set the limit somewhere?